packages/opds-web-client/src/components/Collection.tsx

```diff
@@ -102,24 +101,25 @@ export default class Collection extends React.Component<CollectionProps, any> {
 
   componentWillReceiveProps(nextProps) {
     if (this.props.isFetching && !nextProps.isFetching && !nextProps.error) {
-       document.body.scrollTop = 0;
-       document.documentElement.scrollTop = 0;
+      (this.refs["body"] as any).scrollTop = 0;
```

Comment:
This was needed because document.body is no longer the element that scrolls. I renamed the body ref so this is more clear. I was able to change some of the casts to HTMLElement.